servers/cu/src/domain/client/ao-process.js

```diff
@@ -1267,6 +1267,7 @@ export function saveLatestProcessMemoryWith ({ cache, logger, saveCheckpoint, EA
     // const zipped = await gzipP(Memory, { level: zlibConstants.Z_BEST_SPEED })
     // stopTimer()
 
+    const incrementedGasUsed = (cached?.evaluation?.gasUsed ?? BigInt(0)) + (gasUsed ?? BigInt(0))
```

Comment:
This is a little hard to read. Perhaps you could use a `pipe` with `ramda` utilities, such that it's more declarative? Just a suggestion.